### DISCUSSION TITLE
Database vacuum extremely slow on startup

### DISCUSSION THREAD

**Discussion by YBonline** [blakeblackshear/frigate#13923] [Ask A Question] [answered]
I've gotten a little annoyed about how long Frigate has been taking to start up, which has been growing (probably because I added a few new drives to my array to store recordings longer, I now have about 14TB of recordings) and my database has grown to 30GB. Looking into it, all the startup delay is a result of the database running the vacuum command. I'm curious, is there ways to optimize this? Is it really necessary to vacuum as often or there a way to disable this? Is it supposed to vacuum on every startup or just if its been more then 2 weeks?

I'm on 0.14.1 FYI

**Comment by NickM-27** [comment, marked as answer]
It should only vacuum if it's been more than 2 weeks, but if your config directory is not stored correctly in docker then it doesn't have a way to know when that last occurred.
